README.txt

```diff
@@ -77,8 +77,8 @@ lib [This folder contains 3rd party libraries needed in the project]
 		3 - To compile the test, type in the terminal:
 			make clean; make comp; make all
 		4 - To run the test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
-			./SUBNET
-		5 - To check the output of the test, open  "/test/data/subnetsubnet_test_output.txt"
+			./SUBNET_TEST
+		5 - To check the output of the test, open  "/test/data/subnet_test_output.txt"

```

Comment:
User would need to move back from bin folder after running the executable files. Hence, it would be open “../test/data/subet_test_output.txt”.

Reply:
It has been fixed thankyou for pointing towards it